core/src/main/scala/org/apache/spark/util/SignalLogger.scala

```diff
@@ -17,44 +17,20 @@
 
 package org.apache.spark.util
 
-import org.apache.commons.lang3.SystemUtils
 import org.slf4j.Logger
-import sun.misc.{Signal, SignalHandler}
 
 /**
  * Used to log signals received. This can be very useful in debugging crashes or kills.
- *
- * Inspired by Colin Patrick McCabe's similar class from Hadoop.
  */
 private[spark] object SignalLogger {
```

Comment:
I kept it because logging is called from more than one place (two to be precise)